src/components/rockets/rocket.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchRocketDataFromAPI } from '../../redux/rockets/rocketsReducer';
+import RocketItem from './RocketItem';
+
+function Rockets() {
+  const dispatch = useDispatch();
+  const rocket = useSelector((state) => state.Rockets);
+
+  useEffect(() => {
+    dispatch(fetchRocketDataFromAPI());
+  }, [dispatch]);
```

Comment:
- [ ] When I reserve some rockets and navigate away from the page to maybe the missions page and come back to the rockets page, the reserved rockets are gone. I believe this is because your rockets are fetched every time one visits the rockets page, which causes a re-render, thereby clearing the previous state and using new data from the API.

  Kindly note that the rockets' data should be fetched from the API only once, just like the Missions data.